Jenkinsfile

```diff
@@ -44,6 +44,40 @@ pipeline {
             }
           }
         }
+        stage("Build / Test - JDK19") {
+          agent { node { label 'linux' } }
+          steps {
+            container('jetty-build') {
+              timeout( time: 180, unit: 'MINUTES' ) {
+                checkout scm
+                mavenBuild( "jdk19", "clean install javadoc:javadoc -Perrorprone", "maven3")
+                // Collect up the jacoco execution results (only on main build)
+                jacoco inclusionPattern: '**/org/eclipse/jetty/**/*.class',
```

Comment:
I think we can skip jacoco for this build.

Reply:
Not sure it even works on jdk19 anyway.

Reply:
certainly not for now but not the main concern of this PR :)